pkg/ruler/ruler.go

```diff
@@ -1095,3 +1185,74 @@ func (r *Ruler) ListAllRules(w http.ResponseWriter, req *http.Request) {
 	close(iter)
 	<-done
 }
+
+// NotifySyncRulesAsync enqueue a request to notify other rulers to reload the configuration
```

Comment:
nit: other rulers => all rulers. It also notifies "this ruler", right?

Reply:
Yes, fixed.